test/javascripts/lib/pretty-text-test.js.es6

```diff
@@ -1387,6 +1387,13 @@ QUnit.test("enable/disable features", assert => {
   );
 });
 
+QUnit.test("table inside code block", assert => {
+  assert.cooked(
```

Comment:
Is that testing your change? Isn't the fix related to _pasting_ a table in a code block?

Reply:
I think *real* pasting can't be tested with qunit, no?

Reply:
Also the problem here is that is has also worked before. We're just not wasting resources anymore now.